workflow_scripts/tests/test_mutation_run_mutmut.py

```
@@ -167,18 +249,54 @@ def test_scoped_run_passes_module_globs(
+        )
+        assert "survived" in counts_line, (
+            "the counts diagnostic should carry the parsed status tallies"
+        )
 
     @POSIX_SHIMS_ONLY
     def test_failing_baseline_propagates_exit_code(
-        self, tmp_path: Path, monkeypatch: pytest.MonkeyPatch, fake_uv: Path
+        self,
+        tmp_path: Path,
+        monkeypatch: pytest.MonkeyPatch,
+        fake_uv: Path,
```

Ruff (0.15.21)

[warning] Unused method argument: `fake_uv`

(ARG002)